soh/soh/Enhancements/randomizer/3drando/settings.cpp

```diff
@@ -332,6 +341,14 @@ namespace Settings {
     &GossipStoneHints,
     &ClearerHints,
     &HintDistribution,
+    &AltarHintText,
+    &GanondorfHintText,
+    &DampeHintText,
+    &Kak10GSHintText,
```

Comment:
Missing the `WarpSongHints` in this misc group map, which is the last piece needed to get it to print in the spoiler log.